src/collections/_documentation/platforms/dotnet/wpf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Windows Presentation Foundation
+sidebar_order: 40
+---
+
+Sentry's .NET SDK works with Windows Presentation Foundation applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WPF apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
+
+## Configuration
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WPF specific configuration.
```

Comment:
Suggest (similar to previous edit): "In addition to [configuring the main call SentrySdk.Init](/platforms/dotnet/), you must also configure the WPF specific configuration"